Frontend/ui-customer/src/Flow.purs

```diff
@@ -140,6 +144,7 @@ baseAppFlow (GlobalPayload gPayload) refreshFlow = do
   when (not refreshFlow) $ void $ UI.splashScreen state.splashScreen
   _ <- lift $ lift $ liftFlow $ logEventWithParams logField_ "ny_user_app_version" "version" versionName
   _ <- lift $ lift $ liftFlow $ logEvent logField_ "ny_user_entered_app"
+  void $ pure $ firebaseLogEvent $ getValueToLocalStore REFERRER_URL
```

Comment:
it can be `__failed` as it is async. can you make a new thread and make callback to log event. or call the `firebaseLogEvent` in native itself.